tour/authorizations.md

```diff
@@ -2,7 +2,7 @@
 title: Authorizations
 ---
 
-[Authorizations] are a set of Strings that enable a user to read protected data. Users are granted authorizations
+[Authorizations] are a set of strings that enable a user to read protected data. Users are granted authorizations
```

Comment:
It's probably fine to preserve the uppercase for these, since we literally mean `String` in Java. It might help to use syntax highlighting in Markdown whenever we reference a specific class (singular or plural), such as:

```suggestion
[Authorizations] are a set of `String`s that enable a user to read protected data. Users are granted authorizations
```